lib/mixlib/install/options.rb

```diff
@@ -71,8 +79,7 @@ def validate_channels
       end
 
       def validate_unstable_version
-        if ARTIFACTORY_CHANNELS.include?(channel) &&
-            product_version !~ /^\d+.\d+.\d+\+[0-9]{14}$/
+        if for_artifactory? && product_version !~ /^\d+.\d+.\d+\+[0-9]{14}$/
```

Comment:
Ok.  We did add some version string validation.  I don't recall if we took that upon ourselves or if there was a reason for it.  Looking at it now, it's probably not necessary.